src/test/scala/is/hail/methods/AnnotateSuite.scala

```diff
@@ -467,281 +467,5 @@ class AnnotateSuite extends SparkSuite {
 
     assert(byPosition.same(byVariant))
   }
-
-  @Test def testTypes() {
```

Comment:
the log this is testing is ridiculous and is going to get removed. I really didn't want to rewrite it.

Reply:
Agreed.  I will change the annotateVariantsTable behavior.